src/components/Center/index.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import { css } from 'glamor'
+import PropTypes from 'prop-types'
+import { PullQuote } from '../../components/PullQuote'
+
+export const BREAKOUT_SIZE = {
```

Comment:
IMO not necessary, just pass an array to prop types. It will warn developers if they use an invalid one